blockdevice/blockdevice.go

```diff
@@ -156,6 +159,16 @@ type FileSystemInformation struct {
 	MountPoint []string
 }
 
+// TemperatureInformation stores the temperature information of the blockdevice
+type TemperatureInformation struct {
+	// TemperatureDataValid specifies whether the current temperature
+	// data reported is valid or not
```

Comment:
how do you determine if the temperature value is valid?

Reply:
Seachest gives that information. For every temperature information that seachest provides there is an associated boolean value, which specifies whether the data is valid or not